control/velocity_controller/include/velocity_controller/LQR_setup.hpp

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include <Eigen/Dense>
+#include <geometry_msgs/msg/wrench_stamped.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/string.hpp>
+#include "PID_setup.hpp"
+#include "velocity_controller/utilities.hpp"
+
+/*class Guidance_values{
+    //Dataclass to store guidance values for LQR controller
+    public:
+    double surge=0.0;    double pitch=0.0;    double yaw=0.0;
+    double integral_surge=0.0;    double integral_pitch=0.0;    double
+integral_yaw=0.0;
+};
+*/
+class LQRparameters {
```

Comment:
Would make this a struct (usually call these "POD" structs (Plain Old Data))

Reply:
(Goes for everywhere else as well, not gonna add review comments in the other places)